docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md

```diff
@@ -94,11 +110,6 @@ The next step in setting up LORIS is running the script `install.sh` in the
 This will begin an interactive setup process that will configure files and
 permissions required to get LORIS up and running.
 
-The install script will prompt for the user password in order to use `sudo` permission for configuring
```

Comment:
Why remove this paragraph?   Definitely keep it, is what I'd suggest

Reply:
This was removed in the PR #6181 by John. I believe it was removed because the above addition grants sudo privileges, so these instructions are not necessary. Let me know what you think.